frontend/ui/widget/container/movablecontainer.lua

```diff
@@ -0,0 +1,295 @@
+--[[--
+A MovableContainer can have its content moved on screen
+with Swipe/Hold/Pan.
+Can optionally apply alpha transparency to its content.
+
+With Swipe: the widget will be constrained to screen borders.
+With Hold and pan, the widget can overflow the borders.
+
+Hold with no move will reset the widget to its original position.
+If the widget has not been moved or is already at its original
+position, Hold will toggle between full opacity and 0.7 transparency.
+
+This container's content is expected to not change its width and height.
```

Comment:
While moving or ever?

Reply:
Not sure. I added that comment early, and may be it could work after I worked on the update region - possibly some side effects on the restrict to screen stuff - well, dunno if we have a real size changing widget to test it with :)

Reply:
I don't think we do. There are some things that superficially look like it (e.g., full Wikipedia) but nothing that actually does.